debian/control

```diff
@@ -14,7 +14,7 @@ XS-Testsuite: autopkgtest
 Package: unattended-upgrades
 Architecture: all
 Depends: ${shlibs:Depends}, ${misc:Depends}, debconf, python3, 
- python3-apt, apt-utils, apt, ucf, lsb-release, lsb-base (>= 3.2-14),
+ python3-apt, apt-utils, apt (>= 1.4.2~), ucf, lsb-release, lsb-base (>= 3.2-14),
```

Comment:
This is slightly problematic - I would like unattended-upgrades to be trivially backportable as far as possible. This is why it runs under both py2 and py3 and does not use advanced python-apt features. If we can avoid making this a hard dependencies and instead do runtime detection of whatever it is we need, I would prefer that. I'm not far enough into the PR yet to see why this is needed :)